src/core_plugins/kibana/public/management/sections/indices/_create.js

```diff
@@ -24,7 +24,7 @@ uiModules.get('apps/management')
     name: config.get('indexPattern:placeholder'),
     isTimeBased: true,
     nameIsPattern: false,
-    notExpandable: false,
+    expandable: false,
```

Comment:
`notExpandable` is a property of the index pattern, `expandable` is just a scope variable. There's no two way binding between `expandable` and `notExpandable`. I just changed the name of the scope variable to better reflect the input in the UI, and the double negatives were driving me mad.